commercial/app/views/static/surveySimplePageMembership.scala.html

```diff
@@ -0,0 +1,63 @@
+@(StaticSurveyMembershipPage: controllers.commercial.StaticSurveyMembershipPage)(implicit request: RequestHeader)
+
+@import conf.switches.Switches._
+@import common.InlineJs
+@import templates.inlineJS.blocking.js._
+
+<!DOCTYPE html>
+<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+
+<head>
+    @fragments.head(StaticSurveyMembershipPage, None, Html(""))
+</head>
+    <body id="top" itemscope itemtype="http://schema.org/WebPage">
+
+        <a class="u-h skip" href="#maincontent" data-link-name="skip : main content">Skip to main content</a>
+
+        @fragments.header(StaticSurveyMembershipPage)
+
+        <div class="l-side-margins">
+
+            <div class="live-events-survey-wrapper" data-link-name="live events survey page">
+
+                <div class="live-events-survey-wrapper-inner">
+
+                    @fragments.inlineSvg("survey-thumbsup", "commercial")
+
+                    <h1>You're ahead of your time.</h1>
```

Comment:
Grammatically, for contractions, we should be using an apostrophe - so it more about having the font render apostrophes correctly?